LibCarla/source/carla/road/MapBuilder.cpp

```diff
@@ -194,8 +194,10 @@ namespace road {
       lc = RoadInfoMarkRecord::LaneChange::Decrease;
     } else if (lane_change == "both") {
```

Comment:
```suggestion
    } else if (lane_change == "both") {
```
Did you forget to remove this line ?